Demos/Aspose.Email.Cloud.Live.Demos/Aspose.Email.Cloud.Live.Demos.UI/Scripts/Common/filedrop.js

```diff
@@ -0,0 +1,268 @@
+            var id = 1;
+            var found;
+            do {
+                found = false;
+
+                for (let i = 0; i < droppedFiles.length; i++) {
+                    if (droppedFiles[i].id === id) {
+                        id += 1;
+                        found = true;
+                        break;
+                    }
+                }
+            } while (found);
+            return id;
+        };
+
+        var preventFileDrop = function (evt) {
+            evt = evt || event;
+            evt.preventDefault();
+            evt.stopPropagation();
+        };
+
+        var removeFileBlock = function (id) {
+            var pos;
+            for (pos = 0; pos < droppedFiles.length; pos++) {
+                if (droppedFiles[pos].id === id) {
+                    break;
+                }
+            }
+            if (pos < droppedFiles.length) {
+                droppedFiles.splice(pos, 1);
+                $('#filedrop-' + randomId).find('#fileupload-' + id).remove();
+                if (droppedFiles.length === 0) {
+                    $('#filedrop-' + randomId).find('.chooseFilesLabel').removeClass('hidden');
+                }
+            }
+        };
+
+        var moveUpFileBlock = function (id) {
+            var pos;
+            for (pos = 0; pos < droppedFiles.length; pos++) {
+                if (droppedFiles[pos].id === id) {
+                    break;
+                }
+            }
+            if (pos < droppedFiles.length && pos !== 0) {
+                var prevId = droppedFiles[pos - 1].id;
+                var flTemp = droppedFiles[pos - 1];
+                droppedFiles[pos - 1] = droppedFiles[pos];
+                droppedFiles[pos] = flTemp;
+
+                var block = $('#filedrop-' + randomId + ' > #fileupload-' + id).detach();
+                $('#filedrop-' + randomId + ' > #fileupload-' + prevId).before(block);
+            }
+        };
+
+        var moveDownFileBlock = function (id) {
+            var pos;
+            for (pos = 0; pos < droppedFiles.length; pos++) {
+                if (droppedFiles[pos].id == id) {
+                    break;
+                }
+            }
+            if (pos < droppedFiles.length && pos != (droppedFiles.length - 1)) {
+                var nextId = droppedFiles[pos + 1].id;
+                var flTemp = droppedFiles[pos + 1];
+                droppedFiles[pos + 1] = droppedFiles[pos];
+                droppedFiles[pos] = flTemp;
+
+                var block = $('#filedrop-' + randomId + ' > #fileupload-' + id).detach();
+                $('#filedrop-' + randomId + ' > #fileupload-' + nextId).after(block);
+            }
+        }
+
+        var appendFileBlock = function (file) {
+            var id = nextFileId();
+            var name = file.name;
+            var fileMoveUpLink = null;
+            var fileMoveDownLink = null;
+            if (options.UseSorting) {
+                fileMoveUpLink = $('\
```

Comment:
I think that the [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) should be used here